src/game_objects/process.py

```diff
@@ -62,13 +65,34 @@ def starvation_level(self):
     def display_blink_color(self):
         return self._display_blink_color
 
+    def _update_blocking_condition(self, update_fn):
```

Comment:
was this section recently removed? it may have been re-enabled in my rebase mess

Reply:
It was not removed, but it was moved lower. So it appears to be duplicated now.

Reply:
this duplicate was removed